presidio-image-anonymizer/presidio_image_anonymizer/entities/api_request_convertor.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+from typing import Tuple, Union, io
+
+from PIL import Image
+
+from presidio_image_anonymizer.entities import InvalidParamException
+
+logger = logging.getLogger("presidio-image-anonymizer")
+
+
```

Comment:
We don't have build verification in the PR for image anonymizer, do we have a story for that?
Meanwhile run lint locally, you're missing docstirng here

Reply:
Add flake8 run locally.